### PR TITLE
Rewrite of useLayout to avoid a delayed rendering. Adding render-debouncer.

### PR DESCRIPTION
**Pull request by hrigner**
Always using the render-debouncer in useLayout. Didn't see any advantages of keeping it optional.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,18 +5,31 @@
...
+You can import each hook individually, e.g `import useModel from 'hamus.js/src/use-model'`,
 
-- [`useModel`](./docs/useModel.md) &mdash; creates a session object from a definition.
-- [`useLayout`](./docs/useLayout.md) &mdash; fetches the layout from a model, and updates the layout on model changes.
-- [`usePicasso`](./docs/usePicasso.md) &mdash; renders a [picasso.js](https://github.com/qlik-oss/picasso.js) visualization to an element.
+or you can use ES6 named imports, e.g. `import { useModel } from 'hamus.js'`.
+
+When using ES6 named imports you might run into missing dependency errors from hooks that you don't actually use in your project.
+To resolve these errors, you either have to install the dependencies, or you can tranform the named import statements to individual
+import statements with `[babel-plugin-import](https://github.com/ant-design/babel-plugin-import)`. The `.babelrc` file will look sth
```
Comment (peol) [inline README.md]: `sth` -> something

File: package.json [hunk 2]
```diff
@@ -25,11 +25,11 @@
     "react-use-promise": "0.2.0"
   },
   "peerDependencies": {
-    "picasso.js": "0.25.1",
-    "picasso-plugin-q": "0.25.1",
-    "react": "16.8.6",
-    "react-dom": "16.8.6",
-    "enigma.js": "2.4.0"
+    "picasso.js": "^0.x",
```
Comment (peol) [inline package.json]: I think there's a minimum version of picasso that we need due to how it's exported. @miralemd ?

Comment (miralemd) [reply package.json]: can't remember any major changes in the way it's exported, but I would recommend using `>=0.14.0`

Comment (hrigner) [reply package.json]: Thanks! :)

**Pull request merged**